src/addNewComment.js

```diff
@@ -0,0 +1,22 @@
+const addNewComment = (id) => {
+  class MessageObject {
+    constructor(mealId, username, message) {
+      this.item_id = mealId;
+      this.username = username;
+      this.comment = message;
+    }
+  }
+  const usernameInput = document.querySelector('.adding-name').value;
+  const commentInput = document.querySelector('.adding-comment').value;
+  const newmsg = new MessageObject(id, usernameInput, commentInput);
+
+  fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5OvdDl22dcA0TUpGjcko/comments', {
```

Comment:
- Comments aren't loading kindly work on it:

![Screen Shot 2021-10-14 at 20 26 29](https://user-images.githubusercontent.com/22258274/137417717-7a987987-5e3c-4a6f-89bb-1ccce3cf5c83.png)